tests/interfaces/default_qubit_2_integration/test_jax_jit_qnode_default_qubit_2.py

```diff
@@ -46,19 +51,26 @@
 
 
 @pytest.mark.parametrize(
-    "interface,dev,diff_method,grad_on_execution", interface_and_qubit_device_and_diff_method
+    "interface,dev,diff_method,grad_on_execution,device_vjp",
+    interface_and_qubit_device_and_diff_method,
 )
 class TestQNode:
     """Test that using the QNode with JAX integrates with the PennyLane
     stack"""
 
-    def test_execution_with_interface(self, dev, diff_method, grad_on_execution, interface):
+    def test_execution_with_interface(
+        self, dev, diff_method, grad_on_execution, interface, device_vjp
+    ):
         """Test execution works with the interface"""
-        if diff_method == "backprop":
-            pytest.skip("Test does not support backprop")
+        if dev.name == "param_shift.qubit":
+            pytest.xfail("gradient transforms have a different vjp shape convention")
```

Comment:
is this being tracked elsewhere?

Reply:
I have a technical story for v0.35 for "VJP's have a well defined, consistent shape".